Include.psm1

```diff
@@ -1031,7 +1020,7 @@ function Get-Device {
                             "Advanced Micro Devices" { "AMD" }
                             "Intel" { "INTEL" }
                             "NVIDIA" { "NVIDIA" }
-                            default { $Device_CIM.Manufacturer -replace '\(R\)|\(TM\)|\(C\)' -replace '[^A-Z0-9]' }
+                            default { $Device_CIM.Manufacturer -replace '\(R\)|\(TM\)|\(C\)|Series' -replace '[^A-Z0-9]' }
```

Comment:
This change is rejected and will be removed at 12 noon GMT: 
Random words should not be hardcoded into the function and is not guaranteed to be a complete or correct list. This removes part of the model name and assumes the words have no importance in the model naming now or in the future. The word ‘series’ does not fit the pattern of ‘(R)’, ‘(TM)’ and ‘(C)’ which are recognized ASCII approximations of official symbols that declare rights to a word but are not intended to be words in themselves.